src/Components/NavBar/index.js

```diff
@@ -0,0 +1,4 @@
+import NavBarItem from './NavbarItem/component';
+import NavBar from './NavbarParent/component';
```

Comment:
Rename folders to `NavBar` as well .. As they are named this way in bootstrap and even when you export it as variables. Just add more consistency to everything.

Also probably drop `..parent` part, just `NavBar` or `Nav` as is in `'react-bootstrap/Nav';`